pulp_2to3_migration/app/pre_migration.py

```diff
@@ -247,13 +247,27 @@ async def pre_migrate_lazycatalog(content_type):
                                 pulp2_content_type_id=lce.unit_type_id,
                                 pulp2_storage_path=lce.path,
                                 pulp2_url=lce.url,
-                                pulp2_revision=lce.revision)
+                                pulp2_revision=lce.revision,
+                                is_migrated=False)
         pulp2lazycatalog.append(item)
 
         save_batch = (i and not (i + 1) % batch_size or i == total_lce - 1)
         if save_batch:
             Pulp2LazyCatalog.objects.bulk_create(pulp2lazycatalog, ignore_conflicts=True)
             pulp2lazycatalog = []
+    importer_obj_id = set(
```

Comment:
@goosemania i think we can remove this code, since we are flipping the flag to True here https://github.com/pulp/pulp-2to3-migration/pull/145/files#diff-30eb3d564f2ed0333cc3fccd41652ccfR365
please double check

Reply:
+1 to remove (but save it somewhere :) in case we are wrong and need to come back to it)